examples/evm/call-contract-with-token-express/index.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const {
+    getDefaultProvider,
+    Contract,
+    constants: { AddressZero },
+    ethers,
+} = require('ethers');
+const { deployUpgradable } = require('@axelar-network/axelar-gmp-sdk-solidity');
+const DistributionExecutable = rootRequire(
+    './artifacts/examples/evm/call-contract-with-token-express/DistributionExpressExecutable.sol/DistributionExpressExecutable.json',
+);
+const DistributionProxy = rootRequire(
+    './artifacts/examples/evm/call-contract-with-token-express/DistributionProxy.sol/DistributionProxy.json',
+);
+const Gateway = rootRequire(
+    './artifacts/@axelar-network/axelar-gmp-sdk-solidity/contracts/interfaces/IAxelarGateway.sol/IAxelarGateway.json',
+);
+const IERC20 = rootRequire('./artifacts/@axelar-network/axelar-gmp-sdk-solidity/contracts/interfaces/IERC20.sol/IERC20.json');
+
+async function deploy(chain, wallet) {
+    console.log(`Deploying DistributionExecutable for ${chain.name}.`);
+    const provider = getDefaultProvider(chain.rpc);
+    chain.wallet = wallet.connect(provider);
+    console.log(`Deploying DistributionProxy for ${chain.name}.`);
+    chain.contract = await deployUpgradable(
+        chain.constAddressDeployer,
+        wallet,
+        DistributionExecutable,
+        DistributionProxy,
+        [chain.gateway, chain.gasService],
+        [chain.gateway, '0xfb72239394647e97894585D0D93Ca91f6C3852a4'],
```

Comment:
updated the hardcoded gas service address here once it's available in https://github.com/axelarnetwork/axelar-cgp-solidity/blob/main/info/testnet.json